orchestrator/workflows/utils.py

```diff
@@ -186,7 +186,7 @@ def create_workflow(
     description: str,
     initial_input_form: Optional[InputStepFunc] = None,
     status: SubscriptionLifecycle = SubscriptionLifecycle.ACTIVE,
-    update_ticket_step: Optional[Step] = None,
+    additional_steps: StepList = begin,
```

Comment:
Looks a bit odd for `begin` to be inserted in the middle of the workflow. Perhaps just default to `StepList()`?

Reply:
That is what I initially had 😃, but then ruff started to complain: `B008 Do not perform function call 'StepList' in argument defaults`, but I can off course ignore that warning.

Reply:
Yeah, ruff is right, and I'm wrong. There shouldn't be a list in the argument defaults.

Better to default to `None` and evaluate in the function body with `additional_steps = additional_steps or StepList()`

Reply:
Applied suggestion.